app/api_parser.py

```diff
@@ -0,0 +1,38 @@
+import json
+import requests
+from config import FORECAST_API_KEY, GOOGLE_API_KEY
+
+
+class ForeCast:
+
+    def __init__(self, latitude, longitude):
+        self.url = 'https://api.forecast.io/forecast/{}/{},{}'.format(FORECAST_API_KEY, latitude, longitude)
+
+    def get_request(self):
+        return json.loads((requests.get(self.url)).text)
+
+
+class GeoCode:
+
+    def __init__(self, address):
+        self.url = 'https://maps.googleapis.com/maps/api/geocode/json?address={}&key={}'.format(address, GOOGLE_API_KEY)
```

Comment:
are you sure you don't need to url-encode the address? what if i text in “Akazienstraße 1” ? (note the sharp s)